tests/iter/tyieldintry.nim

```diff
@@ -1,17 +1,15 @@
 discard """
-targets: "c"
-output: "ok"
+  targets: "c cpp"
 """
-var closureIterResult = newSeq[int]()
 
-# XXX Investigate why this fails now for 'nim cpp'
```

Comment:
Does this test still fail for `nim cpp`?

Reply:
https://github.com/nim-lang/Nim/pull/18635/files#diff-e3754feef189a037951027197731a347d17cb4005d7e53aa116574d86a031d43R2

Reply:
I don't understand. Does it still fail when compiled and run using the C++ backend?

Reply:
see the diff, it adds `targets: "c cpp"` so, cpp is now tested